Dockerfile

```diff
@@ -40,16 +44,26 @@ ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
 # golangci-lint
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/bin ${GOLANGCI_VERSION}
 
-# Git identifier to checkout
+# Git identifier to checkout for tnf
 ARG TNF_VERSION
 ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
 ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
 
+# Git identifier to checkout for partner
+ARG TNF_PARTNER_VERSION
+ARG TNF_PARTNER_SRC_URL=https://github.com/test-network-function/cnf-certification-test-partner
+ARG GIT_PARTNER_CHECKOUT_TARGET=$TNF_PARTNER_VERSION
+
 # Clone the TNF source repository and checkout the target branch/tag/commit
 RUN git clone --no-single-branch --depth=1 ${TNF_SRC_URL} ${TNF_SRC_DIR}
 RUN git -C ${TNF_SRC_DIR} fetch origin ${GIT_CHECKOUT_TARGET}
 RUN git -C ${TNF_SRC_DIR} checkout ${GIT_CHECKOUT_TARGET}
 
+# Clone the partner source repository and checkout the target branch/tag/commit
+RUN git clone --no-single-branch --depth=1 ${TNF_PARTNER_SRC_URL} ${TNF_PARTNER_SRC_DIR}
+RUN git -C ${TNF_PARTNER_SRC_DIR} fetch origin ${GIT_PARTNER_CHECKOUT_TARGET}
+RUN git -C ${TNF_PARTNER_SRC_DIR} checkout ${GIT_PARTNER_CHECKOUT_TARGET}
+
 # Build TNF binary
 WORKDIR ${TNF_SRC_DIR}
 # TODO: RUN make install-tools
```

Comment:
Is call to "make local-test-infra" missing here?

Reply:
I just looked and apparently no need to.
The github action for tnf triggers the github action for the partner repo which runs "make install" which applies the yamls.